services/buildkite/buildkite.tester.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const ServiceTester = require('../service-tester');
+const t = new ServiceTester({ id: 'buildkite', title: 'Buildkite Builds' });
+module.exports = t;
+
+t.create('buildkite invalid pipeline')
+  .get('/unknown-identifier/unknown-branch.json')
+  .expectJSON({ name: 'buildkite', value: 'invalid' });
+
+t.create('buildkite valid pipeline')
+  .get('/3826789cf8890b426057e6fe1c4e683bdf04fa24d498885489/master.json')
+  .expectJSON({ name: 'buildkite', value: 'Passing' });
```

Comment:
Could you please also add the following tests:
- valid `identifier`/`id` with invalid `branch`
- [invalid JSON response](https://github.com/badges/shields/blob/master/services/cdnjs/cdnjs.tester.js#L28-L34)
- [connection error](https://github.com/badges/shields/blob/master/services/appveyor/appveyor.tester.js#L28-L31)

Reply:
Having a hard time implementing the `invalid JSON response` one, I don't quite understand what is the linked example doing. Following something similar I get the same responses as connection error back 

```
 + expected - actual

       {
      -  "name": "build"
      -  "value": "inaccessible"
      +  "name": "buildkite"
      +  "value": "invalid"
       }
```

Reply:
[This line](https://github.com/badges/shields/blob/master/services/cdnjs/cdnjs.tester.js#L6) will also be needed for the `Invalid JSON response` test.
Strange that it's returning `inaccessible` though as it should only return that when there is an error response from `request()`

Reply:
Should I be doing something different?

```
t.create('cdnjs (unexpected response)')
  .get('/_.json')
  .intercept(nock => nock('https://badge.buildkite.com')
    .get('/_.json')
    .reply(invalidJSON)
  )
  .expectJSON({name: 'buildkite', value: 'invalid'});
```

🤔